src/SignalR/clients/ts/FunctionalTests/Startup.cs

```diff
@@ -73,17 +77,33 @@ public void ConfigureServices(IServiceCollection services)
                     {
                         OnMessageReceived = context =>
                         {
-                            var signalRTokenHeader = context.Request.Query["access_token"];
+                            var endpoint = context.HttpContext.Features.Get<IEndpointFeature>()?.Endpoint;
```

Comment:
Auth currently runs twice in SignalR (#9216). Perhaps it would be safer to leave that be in preview4. This particular change was the result of the WebSocketMiddleware not having run prior to endpoint routing - consequently the check `context.HttpContext.WebSockets.IsWebSocketRequest` failed.